tests/v1/test_office.py

```diff
@@ -80,8 +80,8 @@ def test_nonexistent_office(self):
         response_content = json.loads(response.data.decode())
         self.assertTrue(response_content['message'] == "Sorry, no such office exists, try again later!")
 
-    def tearDown(self):
-        return super().tearDown()
+    # def tearDown(self):
+    #     return super().tearDown()
         
 if __name__ == "__main__":
     unittest.main()
```

Comment:
Install Pylint to cater to such linting issues

Reply:
Done!